### PR TITLE
[DSCAlarm] Various Fixes and Enhancements

### PR DESCRIPTION
**Pull request by RSStephens**
This is a re-submission of PR [#2057](https://github.com/openhab/openhab2-addons/pull/2057).
 
Fixes:
* Fixed 'DSC Alarm bridge handler not available. Cannot handle command
without bridge.' error upon binding startup
* The binding was not sending all received messages to the channel
panel_message. This is fixed.
* Fixed DSC Alarm command 060 not working.
* Fixed some errors found by the Static Code Analysis Tool

Enhancements:
* Added send_command channel to allow for sending DSC commands.
* Ability to select the protocol type when using the TCPServer bridge.
* Modified and cleaned up README.md.

Signed-off-by: Russell Stephens rsstephens@gmail.com (github:
RSStephens)

### CODE REVIEW

**Comment by martinvw** [comment]
You should add the signed-off-by to your commit :-)

**Comment by RSStephens** [comment]
I've amended the signed-off-by to the commit.  Thanks Martin!

File: addons/binding/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/handler/PanelThingHandler.java [hunk 1]
```diff
@@ -76,7 +84,8 @@ public void updateChannel(ChannelUID channelUID, int state, String description)
                     try {
                         date = sdfReceived.parse(description);
                     } catch (ParseException e) {
-                        logger.error("updateChannel(): Parse Exception occurred while trying to parse date string: {}. ",
+                        logger.error(
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/handler/PanelThingHandler.java]: Make this a warn please

**Comment by RSStephens** [comment]
I submitted the changes.  Thanks Martin!

**Comment by martinvw** [comment]
Thank you!!

**Pull request merged**